### PR TITLE
CANS-1183: styling fix for Conducted By Select Role field

### PR DESCRIPTION
**Pull request by LeonidMarushevskyi**
## JIRA
https://osi-cwds.atlassian.net/browse/CANS-1183

## Description

Conducted By Select role was not aligned properly compare to other Conducted By fields.

## Tests

- [ ] I used TDD to develop the feature
- [ ] I have included unit tests
- [ ] I have included new acceptance tests or modified existing ones
- [ ] I have included other tests

## Types of changes

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring (No behavioral changes)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [x] I ran all my tests locally which were green.
- [x] My code adds no new issues to Code Climate
- [x] I ran all the linters for the project.
- [x] My code is in a stable state ready to be deployable, but not necessarily complete.
- [x] I promise on my honor as a CWDS developer to ensure I don't break the build (or bring donuts if I leave things broken), to check linters, use best practices, to leave the code in better shape than I found it, and to have a good day.

### CODE REVIEW

File: app/javascript/Application/components/Assessment/AssessmentFormHeader/ConductedByRole.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ class ConductedByRole extends PureComponent {
     const id = 'conducted-by-role'
     const labelId = `${id}-label`
     return (
-      <div>
+      <div className={'domain-comment-block'}>
         <Label id={labelId} for={id} className={'conducted-by-input-label'}>
```
Comment (zliuwd) [inline app/javascript/Application/components/Assessment/AssessmentFormHeader/ConductedByRole.js]: Yeah, it better to have a separate CSS block for "AssessmentFormHeader", the class name is not properly here.

**Comment by LeonidMarushevskyi** [comment]
> Blocking at least until somebody explains to me why we are reusing a domain comment class here.

@wmitchell-cambria Refactored

File: app/javascript/Application/components/Assessment/AssessmentFormHeader/ConductedByNameField.js [hunk 2]
```diff
@@ -14,7 +14,7 @@ class ConductedByNameField extends PureComponent {
     const isValid = !value || value.length <= ConductedByNameMaxLength
     const showError = !disabled && !isValid
     return (
-      <div className={'domain-comment-block'}>
```
Comment (wmitchell-cambria) [inline app/javascript/Application/components/Assessment/AssessmentFormHeader/ConductedByNameField.js]: Ah, I missed this one the first time around. Good catch on this second pass!

**Comment by LeonidMarushevskyi** [comment]
Retest this please

**Pull request merged**